seps/1933-workload-identity-federation.md

```diff
@@ -0,0 +1,228 @@
+   claim as described in Section 4 of [OpenID Connect Discovery 1.0](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfig).
+
+3. **Request OpenID Provider Configuration Document**: The authorization server requests the OpenID Provider Configuration Document
+   as described in Section 4.1 of [OpenID Connect Discovery 1.0](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfigurationRequest).
+
+4. **Return OpenID Provider Configuration Document** The OpenID Provider Configuration Document is returned as described in Section 4.2 of [OpenID Connect Discovery 1.0](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfigurationResponse).
+
+5. **Extract `jwks_uri`**: The authorization server validates the OpenID Provider Configuration Document as
+   described in Section 4.3 of [OpenID Connect Discovery 1.0](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfigurationValidation) and extracts the `jwks_uri` configuration information.
+
+6. **Request JWK Set**: The authorization server retrieves the
+   [RFC7517](https://datatracker.ietf.org/doc/html/rfc7517) JWK set
+   containing the issuer's public keys from the TLS protected location specified by
+   `jwks_uri` configuration information.
+
+7. **Return JWK Set**: The JWK set is returned from the TLS protected location.
+
+8. **Validate JWT Authorization Grant**: The authorization server validates the
+   JWT authorization grant according to
+   [RFC 7523 Section 3](https://datatracker.ietf.org/doc/html/rfc7523#section-3),
+   with considerations specified in this document. This includes verifying the
+   signature using the retrieved public keys and validating that the JWT claims
+   meet the authorization server's configured trust and authorization
+   requirements.
+
+9. **Return Access Token**: If JWT validation succeeds, the authorization server
+   issues an access token suitable for accessing the specified MCP server and
+   returns it to the MCP client per
+   [RFC 7523 Section 2.1](https://datatracker.ietf.org/doc/html/rfc7523#section-2.1).
+
+10. **MCP Server Access**: The MCP client makes a JSON-RPC request to the MCP
+    server, including the access token in the `Authorization` header using the
+    `Bearer` authentication scheme as specified in the baseline MCP Authorization
+    specification.
+
+11. **Validate Access Token**: The MCP server validates the access token according to
+    the baseline MCP Authorization specification requirements, including
+    verifying the token was issued for the MCP server as the intended audience.
+
+12. **MCP Server Response**: If the token is valid and the MCP client is authorized, the
+    MCP server processes the request and returns the JSON-RPC response.
+
+A detailed proposal is described at https://github.com/modelcontextprotocol/ext-auth/blob/pieterkas-wif-extension/specification/draft/workload-identity-federation.mdx
```

Comment:
I am a bit confused about the overlap here. Do we have two proposals? Why is one a SEP and another one in `ext-auth`?